app/models/user.rb

```diff
@@ -18,6 +18,7 @@ class User < ApplicationRecord
   scope :sort_by_name, lambda {|direct, direct_email|
                          order(name: direct, email: direct_email)
                        }
+  has_many :microposts, dependent: :destroy
```

Comment:
Em khai báo belong to mà không khai báo has_many thì điều gì sẽ sảy ra

Reply:
nếu không khai báo has_many thì sẽ không có các phương thức tự động để truy cập vào các bản ghi. Thay vào đó mình phải viết scope truy vấn dùng where